test/index.js

```diff
@@ -59,7 +59,7 @@ describe('stand-in', function () {
 
         replace.restore();
 
-        expect(foo.bar).to.deep.equal(foo.bar);
+        expect(foo.bar).to.shallow.equal(foo.bar);
```

Comment:
I actually think it can be removed completely. It's comparing a value against itself.